media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -263,6 +262,8 @@ pkg_postinst() {
 		elog
 		ewarn "${EROOT}/etc/xdg/autostart/pipewire.desktop has been installed. Users of XDG"
 		ewarn "compliant desktops on OpenRC must not manually start pipewire anymore!"
+		ewarn "Users wishing to use PulseAudio must remove the file manually and add the"
+		ewarn "path above to the INSTALL_MASK variable in their make.conf"
```

Comment:
No, `INSTALL_MASK` is not a feature we will recommend in `elog` to use.

1. `elog` should only show the absolute necessary information for user to get a package into a usable state.
2. `elog` message should not repeat with every update, i.e. only show information when user interaction is required. If we would show same `elog` all the time user would ignore them and probably miss new important information once something has changed.